tf2onnx/function/range.py

```diff
@@ -89,13 +89,10 @@ def make_range_non_const(ctx, start, limit, delta, output, name, dtype):
 
     return nodes
 
-def make_range(ctx, start, limit, delta, output, name, dtype):
-    start_node = ctx.get_node_by_output(start)
-    limit_node = ctx.get_node_by_output(limit)
-    delta_node = ctx.get_node_by_output(delta)
-    if start_node.is_const() and limit_node.is_const() and delta_node.is_const():
-        return make_range_const(ctx, start_node, limit_node, delta_node, output, name, dtype)
-    return make_range_non_const(ctx, start, limit, delta, output, name, dtype)
+def make_range(ctx, start, limit, delta, output, scope_name, dtype):
+    if all([ctx.get_node_by_output(n).is_const() for n in [start, limit, delta]]):
+        return make_range_const(ctx, start, limit, delta, output, scope_name, dtype)
```

Comment:
all([ctx.get_node_by_output(n).is_const() for n in [start, limit, delta]])
=>
all(ctx.get_node_by_output(n).is_const() for n in [start, limit, delta])

no need to create a list again.